snowplow/iglu-client-embedded/schemas/com.metabase/indexmodel/jsonschema/1-0-0

```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Search events",
+  "self": {
+    "vendor": "com.example",
+    "name": "indexmodel",
```

Comment:
From [this table](https://www.notion.so/metabase/0df8245abb8f46edbede12065d65e774?v=86c486a9c8904fb0ab086ad389f63f81) it seems like the schema expected by product is just `model`; could you change it to that? It probably can be used for other model-related events in the future as well.